packages/pigment-css-nextjs-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pigment-css/nextjs-plugin",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

Comment:
@brijeshb42 should we remove the pigment packages (Except for `@mui/material-pigment-css`) from this repo? (In a different PR)

If we remove them, then these probably shouldn't be released.

Reply:
Yes. It's already removed in `next` though. So it'll be double the effort to do the same in master. Let's mark Pigment packages as private in master and once we merge `next` back to `master`, it'll remove them anyways.

Reply:
That makes sense. @aarongarciah lets mark them as private and revert the version bump.

Reply:
The 4 Pigment CSS packages have been marked as private and the bump is reverted in all of them.